docs/getting-started.md

```diff
@@ -13,7 +13,7 @@
 You can install `textlint` using npm:
 
 ```sh
-$ npm install -g textlint
+$ npm install -D textlint
```

Comment:
Can you add `npm init` before this command or add "Creat workspace" secion?
Getting Started aim to support begginer user who is also noder.js begginer.

```sh
# Create package.json
$ npm init --yes

# Installed textlint locally
$ npm install --save-dev textlint
```

OR

	# Create new workspace	

	Create `<your-workspace>` directory and create `package.json` file.
	A [package.json](https://docs.npmjs.com/files/package.json "package.json") manage dependencies of packages that include `textlint`.		
	
	```sh
	# Create your workspace direactory and move it
	mkdir your-workspace
	cd your-workspace

	# `npm init` command create `package.json` file
	npm init --yes
	```
	
	## Installation
	
	...
	
	```sh
	# Installed textlint locally
	$ npm install --save-dev textlint
	```